Assets/MRTK/Services/InputSystem/FocusProvider.cs

```diff
@@ -1098,8 +1121,8 @@ private void UpdatePointer(PointerData pointerData)
                         IMixedRealityPointer gazePointer = InputSystem?.GazeProvider.GazePointer;
                         if (gazePointer.IsNotNull() && pointerData.Pointer.PointerId == gazePointer.PointerId)
                         {
-                            gazeHitResult = hit;
-                        }
+                            gazeHitResult.CopyFrom(hit);
+						}
```

Comment:
A bit of a style nit, but looks like some of this indentation was changed from spaces to tabs. Could it be reverted to spaces please?

Reply:
Have untabified, sorry didn't even realise that'd happened

Reply:
No worries! Thanks for the fix!